robotiq_s_model_control/include/robotiq_s_model_control/s_model_ros.h

```diff
@@ -0,0 +1,65 @@
+#ifndef S_MODEL_ROS_H
+#define S_MODEL_ROS_H
+
+#include <ros/ros.h>
+#include <std_srvs/Trigger.h>
+#include <robotiq_s_model_control/SModel_robot_input.h>
+#include <robotiq_s_model_control/SModel_robot_output.h>
+#include <robotiq_s_model_control/s_model_api.h>
+#include <dynamic_reconfigure/server.h>
+#include <robotiq_s_model_control/SModelConfig.h>
+
+namespace robotiq_s_model_control
+{
+class SModelROS
+{
+public:
+    SModelROS(ros::NodeHandle& nh,
+                  boost::shared_ptr<robotiq_s_model_control::SModelAPI> driver, std::vector<std::string> joint_names,
+                  ros::Duration desired_update_freq);
+
+    void publish();
+
+    bool handle_init(std_srvs::TriggerRequest &req, std_srvs::TriggerResponse &resp);
```

Comment:
I try not to be picky with naming conventions, but this class' public API uses snake case in contrast to the ROS style guide and all the other classes added here. Please convert to camel case.